src/devicelock/devicelock.cpp

```diff
@@ -83,6 +120,27 @@ void DeviceLock::handleCallStateChange(const QString &state, const QString &igno
     }
 }
 
+void DeviceLock::handleBlankingPauseChange(const QString &state)
+{
+    if (state.startsWith("active") && !m_blankingPause) {
```

Comment:
Is there some reason to use startsWith() instead of equality check?

Also I found the double if structure to require some thought to grok. Could be simpler and smaller - something like this perhaps?

    bool blankingPause = (state == "active");
    if (m_blankingPause != blankingPause ) {
      m_blankingPause = blankingPause;
      emit blankingPauseChanged();
    }